pwm/configfile.sls

```diff
@@ -0,0 +1,65 @@
+get_pwm_config:
+  file.managed:
+    - name: /usr/share/tomcat/webapps/ROOT/WEB-INF/PwmConfiguration.xml
+    - source: s3://{{ salt.pillar.get('pwm:lookup:config_bucket') }}/PwmConfiguration.xml
+    - skip_verify: True
+    - user: tomcat    
+    - group: tomcat
+    - mode: 600
+
+get_sasl_password:
+  file.managed:
+    - name: /etc/postfix/sasl_passwd
+    - source: s3://{{ salt.pillar.get('pwm:lookup:config_bucket') }}/sasl_passwd
+    - skip_verify: True
+    - mode: 600
+
+config_start_tomcat_service:
+  service.running:
+    - name: tomcat
+    - enable: True
+    - reload: True
+    - init_delay: 3
+
+create_pwm_config_mgmt_script:
+  file.managed:
+    - name: /usr/local/bin/pwmconfmgmt.sh
+    - source: salt://files/configfile/pwmconfmgmt.sh
+    - template: jinja
+    - mode: 700
+
+create_inotify_pwm_config_script:
+  file.managed:
+    - name: /usr/local/bin/inotifypwmconfig.sh
+    - source: salt://files/configfile/inotifypwmconfig.sh
+    - mode: 700
+
+execute_inotify_script:
+  cmd.run:
+    - name: at now + 20 minutes -f /usr/local/bin/inotifypwmconfig.sh
+
+get_postfix_config_script:
+  file.managed:
+    - name: /usr/local/bin/postfix_conf.sh
+    - source: s3://{{ salt.pillar.get('pwm:lookup:config_bucket') }}/postfix_conf.sh
```

Comment:
I'm a bit curious what is going on in this script. Other files we retrieve from S3 look more like a config value of some kind, rather than an arbitrary script... This is something a _user_ needs to host and provide, how do they know what this ought to do?

Reply:
@ewierschke - do you have an answer for this?

Reply:
Does this config file change enough that it needs to be stored in S3 or is this something we can move into the formula itself?

Reply:
Can be moved to the formula itself

Reply:
@lorengordon and @ewierschke - looking at this file again and it can not go into the formula as it is. That file is storing the credentials (akid/secret key) for SES

Reply:
Ok, this has been fixed